control_rate : 100.0
target_vel_lin : 0.1
w_limit : 1.0


number_of_robots : 1
## topics

robot0_plan_topic : /mir_path

# robot0_trajectory_topic : /mir_trajectory
robot0_trajectory_topic : mir_trajectory

# robot0_cmd_vel_topic: /mur/mir/mobile_base_controller/cmd_vel
robot0_cmd_vel_topic: mobile_base_controller/cmd_vel

# robot0_pose_topic : mur/mir/robot_pose
robot0_pose_topic : amcl_pose
#robot1_pose_topic : /robot1/robot_pose
#robot2_pose_topic : /robot2/robot_pose
#robot0_pose_topic: /amcl_pose
